Makefile

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
That's a good point! Personally, I prefer to have both methods since we use this makefile also in the GitHub actions... using makefile is clearer you just type the exact command we want, I consider it as the real source of truth, I added the `pre-commit` framework to help people who don't want to install this tool by their hands but I don't like too much the approach since you have no full control as in makefile. Anyway we can discuss about that, maybe someone else has some opinion on this :thinking:

Reply:
I thought on it a little bit and we could also use the pre-commit framework in the CI, at the end it's almost the same thing :thinking:  What do you think about it @leogr @deepskyblue86 @FedeDP?